apps/main/e2e/header.spec.ts

```
@@ -0,0 +1,36 @@
+
+	test('should open settings modal from user menu', async ({ posPage: page }) => {
+		await page.getByRole('button', { name: /Demo Cashier/i }).click();
+		await page.getByText('Settings').first().click();
+		await expect(page.getByText(/General|Tax|Barcode|Keyboard|Theme/).first()).toBeVisible({
+			timeout: 10_000,
+		});
+	});
+});
+
+test.describe('Upgrade Banner (Free)', () => {
+	test.beforeEach(async ({}, testInfo) => {
```

Biome (2.3.13)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)